okapi-core/src/main/java/org/folio/okapi/service/impl/PostgresHandle.java

```diff
@@ -57,8 +57,7 @@
         logger.warn("Bad postgres_port value: {}: {}", val, e.getMessage());
       }
     }
-    connectOptions.setUser(Config.getSysConf("postgres_username",
-        Config.getSysConf("postgres_user", "okapi", new JsonObject()), conf));
+    connectOptions.setUser(Config.getSysConf("postgres_username", "postgres_user", "okapi", conf));
```

Comment:
Any reason we need to support both `postgres_username` and `postgres_user`?

Reply:
No good reason. `postgres_user` should die